test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -0,0 +1,371 @@
+Feature: Bind an application to a service using annotations
+
+    As a user of Service Binding Operator
+    I want to bind applications to services it depends on
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+        * PostgreSQL DB operator is installed
+
+    Scenario: Provide binding info through backing service CRD annotation
+        Given The Custom Resource Definition is present
+            """
+                apiVersion: apiextensions.k8s.io/v1beta1
+                kind: CustomResourceDefinition
+                metadata:
+                  name: backends.stable.example.com
+                  annotations:
+                      service.binding/host: path={.spec.host}
+                      service.binding/ready: path={.status.ready}
+                spec:
+                  group: stable.example.com
+                  versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                  scope: Namespaced
+                  names:
+                    plural: backends
+                    singular: backend
+                    kind: Backend
+                    shortNames:
+                      - bk
+            """
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+            spec:
+                host: example.common
+            status:
+                ready: true
+            """
+        * Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-backend-a
+            spec:
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
+                    id: SBR
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "False"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding "binding-request-backend-a" should be changed to "EmptyApplication"
+        And Secret "binding-request-backend-a" contains "BACKEND_READY" key with value "true"
+        And Secret "binding-request-backend-a" contains "BACKEND_HOST" key with value "example.common"
+
+    Scenario: Each value in referred map from service resource gets injected into app as separate env variable
+        Given OLM Operator "backend" is running
```

Comment:
If we have the steo `olm operatoy backend is running` then the yaml in test/resources/backend.operator.manifest.yaml is applied and then we can omit the succeeding step of `The Custom Resource Definition is present`

Reply:
Good point @Avni-Sharma, thx. Actually the CRD (not the olm operator) is essential in those secenarios, so the `olm operator backend is running` can be omiited.

Reply:
Done